src/main/resources/application.properties

```diff
@@ -0,0 +1,9 @@
+server.port = 8080
+
+#DATABASE
+spring.datasource.url=jdbc:postgresql://localhost:5432/rentapark
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.datasource.username=postgres
+spring.datasource.password=postgres
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.hibernate.ddl-auto=update
```

Comment:
later on we can introduce flyway or liquibase, this can stay like this for now but we should add ticket to backlog